logrotation/logrotate/log_rollover.conf

```diff
@@ -0,0 +1,8 @@
+<COMPONENT_LOG_PATH>/*.log {
```

Comment:
Add Note here to adjust configs as per component needs. same for any other config file in this PR

Reply:
done 👍